tests/cases/standalone/common/function/json/json.result

```diff
@@ -36,15 +36,15 @@ SELECT json_path_exists(parse_json('[1, 2]'), 'null');
 +-----------------------------------------------------------+
 | json_path_exists(parse_json(Utf8("[1, 2]")),Utf8("null")) |
```

Comment:
Consider adding a test for NULL value. (Not json "null")

Reply:
Added.